cmd/vic-machine/validate/validator.go

```diff
@@ -130,25 +140,25 @@ func (v *Validator) getConnection(input *data.Data) error {
 		PoolPath:       v.ResourcePoolPath,
 	}
 
-	v.Session, err = session.NewSession(sessionconfig).Create(v.Context)
-	if err != nil {
-		log.Errorf("Failed to create session: %s", err)
-		return err
-	}
+	v.Session = session.NewSession(sessionconfig)
+
 	if _, err = v.Session.Connect(v.Context); err != nil {
-		return err
+		return false, errors.Errorf("%s\nFailed to connect. Verify --target, --user, and --password", err.Error())
 	}
-	return nil
+
+	if _, err = v.Session.Populate(v.Context); err != nil {
```

Comment:
If you move the suggest method to getResources method, you don't need to populate session here. This one should not be used.